packages/vector/collections.py

```
@@ -0,0 +1,303 @@
+from qdrant_client.http import models
+from qdrant_client.http.exceptions import UnexpectedResponse
+
+from packages.common.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+class CollectionCreationError(Exception):
+    """Raised when collection creation fails (excluding 409 conflicts)."""
+
+    pass
```

Ruff (0.14.1)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)